mucommander-translator/src/main/resources/dictionary_ko.properties

```diff
@@ -128,6 +130,8 @@ CheckForUpdates.label = 업데이트 확인
 CompareFolders.label = 폴더 비교
 ConnectToServer.label = 서버에 연결
 ConnectToServer.tooltip = 원격 서버에 연결
+Find.label = $[find]
+View.label = $[view]
```

Comment:
there's not problem with specifying those but it's not really needed - if these entries are not found in the dictionary file for the language, it's taken from dictionary.properties